test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1604,42 +1572,46 @@ def tearDown(self):
         # Disable the cgroups hook
         conf = {'enabled': 'False', 'freq': 30, 'event': events}
         self.server.manager(MGR_CMD_SET, HOOK, conf, self.hook_name)
-        # Cleanup the frozen jobs
+        # Cleanup frozen jobs
+        self.logger.info("Cleaning up frozen jobs ****")
         fdir = self.paths['freezer']
         if os.path.isdir(fdir):
+            print "freezer directory presents"
```

Comment:
Use self.logger() instead of print.  Maybe phrase this as 'freezer directory is present'